interface/patient_tracker/patient_tracker.php

```diff
@@ -175,252 +176,186 @@
-                            <div class="form-group row justify-content-center">
-                                <div class="col-xl-5 col-lg-9 col-md-11 mb-xl-0 mb-1">
-                                    <select id="form_apptcat" name="form_apptcat" class="form-control form-control-sm" onchange="refineMe('apptcat');" title="">
-                                        <?php
-                                        $categories = fetchAppointmentCategories();
-                                        echo "<option value=''>" . xlt("Visit Categories") . "</option>";
-                                        while ($cat = sqlFetchArray($categories)) {
-                                            echo "<option value='" . attr($cat['id']) . "'";
-                                            if ($cat['id'] == $_POST['form_apptcat']) {
-                                                echo " selected='true' ";
-                                            }
-                                            echo ">" . xlt($cat['category']) . "</option>";
-                                        }
-                                        ?>
-                                    </select>
-                                </div>
-
-                                <div class="col-xl-5 col-lg-9 col-md-11">
-                                    <select id="form_apptstatus" name="form_apptstatus" class="form-control form-control-sm" onchange="refineMe();">
-                                        <option value=""><?php echo xlt("Visit Status"); ?></option>
+    <div class="container mt-3">
+        <div class="row" id="flb_selectors" style="display:<?php echo attr($setting_selectors); ?>;">
+            <div class="col-12">
+                <h2 class="text-center"><?php echo xlt('Flow Board'); ?></h2>
+            </div>
+            <div class="col-12 jumbotron p-1">
+                <div class="showRFlow text-center" id="show_flows" name="kiosk_hide">
+                    <div name="div_response" id="div_response" class="nodisplay"></div>
+                        <?php
+                        if ($GLOBALS['medex_enable'] == '1') {
+                            $col_width = "3";
+                        } else {
+                            $col_width = "4";
+                            $last_col_width = "nodisplay";
+                        }
+                        ?>
+                        <form name="flb" id="flb" method="post">
+                            <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                            <div class="text-center row align-items-center">
+                                <div class="col-sm-<?php echo attr($col_width); ?> text-center">
+                                    <div class="form-group row justify-content-center">
+                                        <div class="col-xl-5 col-lg-9 col-md-11 mb-xl-0 mb-1">
```

Comment:
Grid columns shouldn't require this many grid classes as it can mess with responsiveness.